install_ddev.sh

```diff
@@ -49,10 +131,26 @@ else
 fi
 
 if which brew &&  [ -f `brew --prefix`/etc/bash_completion ]; then
-	bash_completion_dir=$(brew --prefix)/etc/bash_completion.d
+    bash_completion_dir=$(brew --prefix)/etc/bash_completion.d
     cp /tmp/ddev_bash_completion.sh $bash_completion_dir/ddev
     printf "${GREEN}Installed ddev bash completions in $bash_completion_dir${RESET}\n"
     rm /tmp/ddev_bash_completion.sh
 fi
 
-printf "${GREEN}ddev is now installed. Run \"ddev\" to verify your installation and see usage.${RESET}\n"
+mkdir -p ~/Sites/sprint
+cp start_sprint.sh ~/Sites/sprint/
+cp sprint.tar.xz ~/Sites/sprint/
+
+printf "
```

Comment:
Yes, sorry, this is where I mean to comment re: it printing out success regardless of if it failed because of my impatience.

Reply:
have added a wait before the echo